website/screens/components/alert/code/AlertCodePage.tsx

```diff
@@ -16,22 +16,26 @@ const sections = [
         <thead>
           <tr>
             <th>Name</th>
+            <th>Type</th>
+            <th>Description</th>
             <th>Default</th>
-            <HeaderDescriptionCell>Description</HeaderDescriptionCell>
           </tr>
         </thead>
         <tbody>
           <tr>
-            <td>type: 'info' | 'confirm' | 'warning' | 'error'</td>
+            <td>type</td>
             <td>
-              <Code>'info'</Code>
+              <TableCode>'info' | 'confirm' | 'warning' | 'error'</TableCode>
             </td>
             <td>Uses on of the available alert types.</td>
```

Comment:
Should be 'Uses **one** of the available alert types' instead of 'Uses on of the available alert types'?